Assets/MixedRealityToolkit.Tests/PlayModeTests/StandardShaderTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class StandardShaderTests : BasePlayModeTests
+    {
+        private const float VisualizatonWaitTime = 1.0f;
+        private readonly Vector3 LightDirection = new Vector3(-0.5f, -0.5f, 0.0f);
+
+        /// <summary>
+        /// Tests if the MeshOutline component can be added an manipulated at runtime.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator TestMeshOutline()
+        {
+            var light = InstantiateDirectionalLight(LightDirection);
```

Comment:
I don't understand what this is testing? Just no errors thrown? There are no asserts

Reply:
Yeah I assume he is just checking that no null ptrs are thrown. I considered suggesting doing screenshot comparison but that seemed like it could break often and also could be error prone.

Reply:
Yep @julenka is correct. We've had so many components that just fall over when instantiated at runtime with null references that this is a sanity check for that. Would love to do screenshot comparisons. But, that would require some work on the test and build machine side. :(